### ISSUE TITLE
RichMessage with reply method

### ISSUE THREAD

**Issue by ram-g-athreya** [ramswaroop/jbot#59]
Hi

Is there a way to use RichMessage with the bot's reply method? The example does not seem to deal with this. Basically I want to show a RichMessage as a response to a Text Message from the user.

Thanks

**Comment by ramswaroop** [comment]
Hi, I think Slack doesn't support RichMessage in RTM api but supports via its web api. You can see `SlashCommand.java` inside `jbot-example` to get an idea on how to use it.

**Comment by rahulsom** [comment]
I think it would help to be able to receive a message over RTM, and then _reply_ over its web api by sending a new message over https://api.slack.com/methods/chat.postMessage.

**Comment by ramswaroop** [comment]
Yes @rahulsom, that would be nice. I shall work on the web api feature later as I am finding it hard to take out time for this now. Having said that, a PR is always welcome.
